### ISSUE TITLE
[Members] Update position holders of communities

### ISSUE THREAD

**Issue by EmperorYP7** [houseofgeeks/hg#227]
### Description
<write your description> Since the community co-ordinators and heads of each community have been updated, it is necessary to reflect that in the website.

### Assets and elaboration

To be updated soon.

### Concerning files/folders

https://github.com/houseofgeeks/hg/tree/front-end/client/src/components/Members/Community

**Comment by tusharjain0022** [comment]
### Assets required for this issue -
#### Competitive Coding Members - 
1. Sahil Gupta (Community Coordinator):
   photo - https://drive.google.com/file/d/1AHny-tEgzohH3m-y2fVOnYmDghN10lKh/view?usp=sharing
   email id - sahilgupta.btech.cs19@iiitranchi.ac.in
   linkedin - https://www.linkedin.com/in/sahil-gupta-765839122/
   github - https://github.com/Sahil-Gupta582

2. Saheb Kumar (Community Coordinator) : 
   photo - https://drive.google.com/file/d/18jz-AhLL0HzV2x9uctGFvyg7d3oRXu9L/view?usp=sharing
   email id - Saheb.btech.cs19@iiitranchi.ac.in
   linkedin - https://www.linkedin.com/in/saheb-kumar-1a1236192
   github - https://github.com/sahebcse

#### Web and App Community Members
1. Yash Pandey (Community Coordinator):
   photo - https://drive.google.com/file/d/1A2vHz14KmXG9ChbvKVqcp0kxWN6SDiqB/view?usp=sharing
   email id - yash.btech.cs19@iiitranchi.ac.in
   linkedin - https://www.linkedin.com/in/emperoryp
   github - https://github.com/EmperorYP7

2. Tushar Jain (Community Coordinator): 
   photo - https://drive.google.com/file/d/1vEl2gXwl9D3mH11rtQ_1pchWSjL9HeUA/view?usp=sharing
   email id - tusharjain.btech.cs19@iiitranchi.ac.in
   linkedin - https://www.linkedin.com/in/tusharjain0022/
   github -https://github.com/tusharjain0022


#### Core Electronics Community Members [update : 8-04-2021]
1. Shashwat Pratap (Community Coordinator):
   photo - https://drive.google.com/file/d/1Rq-8IvWxu9YYcnXyv7shZ1gpiGoW4TVl/view?usp=sharing
   email id - shaswat.btech.ec19@iiitranchi.ac.in
   linkedin - https://www.linkedin.com/in/shashwat-pratap-470b7116b
   github - https://github.com/shashwat0426

PS: I'll add details of the Community Coordinator of Core Electronics later on till then @EmperorYP7 could you verify these details and can edit something if needed.

**Comment by SrijanSriv** [comment]
> #### Assets required for this issue 

Are these all the members that have to be appended for the site? I noticed that we formerly had Community Heads too and we don't anymore.
This will probably mean that a little tweaking would be required style wise.
@tusharjain0022

**Comment by tusharjain0022** [comment]
@EmperorYP7 @SrijanSriv  I think we need to reopen this issue as new Community leaders joined the House of Geeks.

**Comment by SrijanSriv** [comment]
@tusharjain0022 agreed. But I feel like the batch prior to mine hasn't been up for too long. So I was hoping to reopen this issue once the next batch joins so the transition is consistent like the shift of batches(although we are already promoted to the next batch, still).

**Comment by EmperorYP7** [comment]
>  I think we need to reopen this issue as new Community leaders joined the House of Geeks.

That's right

**Comment by Abhinav200207** [comment]
I would like to work upon this issue.

**Comment by SrijanSriv** [comment]
this issue should have been closed after @Abhinav200207 merged his pr (thanks @Abhinav200207 ) but we shall continue to yet again update the position holders. @abhishekrathi16 requested assignment of this issue.

**Comment by abhishekrathi16** [comment]
I'll be taking up the issue then, kindly provide the new files required for updating the placeholders @SrijanSriv

**Comment by SrijanSriv** [comment]
dear abhishek
go fish
🙂
